cm2350/peripherals/dspi.py

```diff
@@ -7,7 +7,7 @@
 from ..intc_exc import INTC_EVENT
 
 import logging
-import envi.common as e_cmn
+from envi.common import EMULOG
```

Comment:
as long as you understand that EMULOG is a vomitous level.  It's created for Emulation pass, specifically for "isProbablyCode()" which makes lots of ugly messages.  This is *more* DEBUG than DEBUG.

Reply:
The Viv maintainers are discussing merging this with MIRE or SHITE :)

Reply:
yeah the intention was to not have these logs be part of standard DEBUG, but they can still be useful. I can put them into MIRE or SHITE if it's preferred?